src/Workleap.AspNetCore.Authentication.ClientCredentialsGrant/RequireClientCredentialsAttribute.cs

```diff
@@ -7,19 +7,46 @@ namespace Workleap.AspNetCore.Authentication.ClientCredentialsGrant;
 [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method)]
 public sealed class RequireClientCredentialsAttribute : AuthorizeAttribute
 {
+    /// <summary>
+    /// asd
+    /// </summary>
     private static readonly Dictionary<ClientCredentialsScope, string> EnumScopeNameMapping = new()
     {
         [ClientCredentialsScope.Read] = "read",
         [ClientCredentialsScope.Write] = "write",
         [ClientCredentialsScope.Admin] = "admin",
     };
 
+    /// <summary>
+    /// Verifies that the web API is called with the right classic scope. <br/>
+    /// - It will check in those claims type: scope, scp or http://schemas.microsoft.com/identity/claims/scop <br/>
```

Comment:
Missing 'e' at the end of `http://schemas.microsoft.com/identity/claims/scop` (3 occurrences)